cpp/comms/std/CMakeLists.txt

```diff
@@ -43,8 +43,10 @@ set(COMMS_LINK_LIBRARIES ${CUML_CPP_TARGET})
 
 # Note this option will be removed once UCX conda package is released
 if(WITH_UCX)
+    # dlopen is used to dynamically load the needed ucp symbols at runtime. 
+    # Calling find_package here just allows an error to be thrown at build 
+    # time if it's not available.
```

Comment:
I'm not entirely sure this comment is accurate, it seems that `find_package(UCX)` is needed to use the `${UCX_INCLUDE_DIRS}` variable and not only to be a build time check

Reply:
That is true. More explicitly, it's important that folks building from source understand why the `find_package` is being called but the library dirs are never used. I'll make this note more explicit.